scripts/jestGlobalSetup.js

```diff
@@ -0,0 +1,20 @@
+const os = require('os')
+const fs = require('fs-extra')
+const path = require('path')
+const { chromium } = require('playwright-chromium')
+
+const DIR = path.join(os.tmpdir(), 'jest_playwright_global_setup')
+
+module.exports = async () => {
+  const browserServer = await chromium.launchServer({
+    headless: !process.env.FR_DEBUG_SERVE,
+    args: process.env.CI
+      ? ['--no-sandbox', '--disable-setuid-sandbox']
+      : undefined
+  })
+
+  global.__BROWSER_SERVER__ = browserServer
+
+  await fs.mkdirp(DIR)
+  await fs.writeFile(path.join(DIR, 'wsEndpoint'), browserServer.wsEndpoint())
+}
```

Comment:
I am ok with this for now, but it does mean that we have to wait for chrome to start for running plain old unit tests too. Maybe there's a way we can only invoke this for the tests that need it, but we can cross that bridge later if it gets really annoying.

Reply:
You're right, I think we can separate the setup of the environment for unit/e2e tests and invoke this only for tests that need it.
Will circle back to it later and make the change.